app/controllers/publishers/policy_agreements_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Publishers::PolicyAgreementsController < ApplicationController
```

Comment:
I think we should add protections for this controller because right now you don't have to be signed in to request this route